test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/CSharpEndToEndTests.cs

```diff
@@ -218,12 +218,12 @@ public void FileLogging_Succeeds()
             Assert.Equal(expectedValue, Utility.RemoveUtf8ByteOrderMark(result));
         }
 
-        [Theory(Skip = "Not yet enabled.")]
-        [InlineData("application/json", "\"Name: Fabio Cavalcante, Location: Seattle\"")]
-        [InlineData("application/xml", "<string xmlns=\"http://schemas.microsoft.com/2003/10/Serialization/\">Name: Fabio Cavalcante, Location: Seattle</string>")]
-        [InlineData("text/plain", "Name: Fabio Cavalcante, Location: Seattle")]
-        public async Task HttpTrigger_GetWithAccept_NegotiatesContent(string accept, string expectedBody)
-        {
+        //[Theory(Skip = "Not yet enabled.")]
```

Comment:
Created https://github.com/Azure/azure-functions-host/issues/2663 to track this.